### ISSUE TITLE
Error encountered while testing a base model.

### ISSUE THREAD

**Issue by efirdc** [brain-score/model-tools#57]
Hello,

I have encountered an error while testing a `base_models.py` implementation. Here is the [log](https://github.com/brain-score/brain-score/files/7424208/error.log) and the [line](https://github.com/brain-score/model-tools/blob/b5c7f300dc20fa9d7c338ee0d930524ea6e7e2bf/model_tools/activations/core.py#L175) where it breaks. There is a comment there that may be describing the issue, but I'm not sure how to interpret it. 

It looks like this traces back to the behavioral benchmark, specifically [here](https://github.com/brain-score/model-tools/blob/b5c7f300dc20fa9d7c338ee0d930524ea6e7e2bf/model_tools/brain_transformation/behavior.py#L44). Is this assuming the model has a layer named `'logits'`? The model I am testing does not have this.

Any advice on how to debug this would be very appreciated.

Thank you,
Cory
